src/assets/google-script/google-script.spec.ts

```
@@ -1,43 +1,312 @@
+    });
+
     TestBed.configureTestingModule({
-      providers: [],
-      imports: []
+      providers: [
+        GoogleScript,
+        {
+          provide: NgZone,
+          useValue: {
+            // eslint-disable-next-line @typescript-eslint/ban-types
+            runOutsideAngular: (fn: Function) => fn(),
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)